locale/en.yml

```diff
@@ -886,6 +887,7 @@ en:
       OrchestrationTemplateVnfd: VNF Template
       ManageIQ::Providers::Vmware::CloudManager::OrchestrationTemplate: vApp Template
       PersistentVolume:         Persistent Volume
+      PhysicalServer:           Physical Server
```

Comment:
https://github.com/ManageIQ/manageiq/pull/15275 is a dup of this PR (merged yesterday)

Although I might have missed adding the entry on L#890. 
Do we need it? (since I did not see it for Physical Infrastructure Provider)
If yes, let's keep the PR. Thanks.